### PR TITLE
update readme with more instructions

### PR DESCRIPTION
**Pull request by elmiko**
includes how to build and some status about the operator and supported
platforms.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,106 @@
...
+on pull requests.
+
+#### Local Unit Tests
+
+To invoke the unit tests, run `make unit`. If you wish to also run the
+code generation and verification steps, run `make test`.
+
+*Note When running the local unit tests you may see errors about a missing
+`etcd` binary. These tests expect a slightly different configuration and
+are known to fail on local hosts.*
```
Comment (JoelSpeed) [inline README.md:96]: Are these kubebuilder tests? Can we link to any instructions on how to set up the etcd binary?

Comment (elmiko) [reply README.md:96]: yeah, i would love to figure this out. any link?

Comment (JoelSpeed) [reply README.md:96]: Maybe something like https://book.kubebuilder.io/reference/envtest.html?

Comment (elmiko) [reply README.md:96]: thanks! i'll investigate a little more

**Pull request merged**